src/libc/include/stdarg.h

```diff
@@ -3,10 +3,10 @@
 
 #include <endian.h>
 
-typedef char *va_list;
-#define va_start(ap, parmn) (void)((ap) = (char *)(&(parmn) + 1))
-#define va_end(ap) (void)((ap) = 0)
-#define va_arg(ap, type) (((type *)((ap) = ((ap) + sizeof(type))))[-1])
+typedef __builtin_va_list va_list;
+#define va_start(ap, param) __builtin_va_start(ap, param)
+#define va_end(ap)          __builtin_va_end(ap)
+#define va_arg(ap, type)    __builtin_va_arg(ap, type)
```

Comment:
Do you want to submit this as a patch to E-ACSL where I believe the non-builtin version is hardcoded in `eacsl.h` ?

Reply:
Yes, the non-builtin MACROs are in `stdarg.h` but I believe they should at least adjust them according to architecture-dependent configuration ?

Reply:
Yes, you can simply change their code to use `stdarg.h` :)